components/clarity-repl/src/repl/debug/cli.rs

```diff
@@ -129,7 +129,6 @@ impl CLIDebugger {
                         "{}:{}:{}",
                         contract_id, expr.span.start_line, expr.span.start_column
                     );
-                    return;
```

Comment:
That seems suspect? Why is the `return` removed here?

Reply:
There is no other code path after this `None` branch